cdap-api/src/main/java/co/cask/cdap/api/flow/FlowletDefinition.java

```diff
@@ -57,20 +51,14 @@
   private Map<String, Set<Schema>> inputs;
   private Map<String, Set<Schema>> outputs;
 
-  public FlowletDefinition(String flowletName, Flowlet flowlet, int instances) {
-    FlowletSpecification flowletSpec;
-    DefaultFlowletConfigurer flowletConfigurer = new DefaultFlowletConfigurer(flowlet);
-    flowlet.configure(flowletConfigurer);
-    flowletSpec = flowletConfigurer.createSpecification();
-
+  public FlowletDefinition(String flowletName, Flowlet flowlet,
```

Comment:
It shouldn't need the `Flowlet` anymore

Reply:
updated to pass in `inputTypes` and `outputTypes` map